scripts/install_backend_python_libs.py

```diff
@@ -115,20 +137,24 @@ def _remove_metadata(library_name, version_string):
     """
     possible_directory_names = _get_possible_metadata_directory_names(
```

Comment:
possible_normalized_directory_names

Again, treat the orig thing and the normalized thing as different concepts, and make sure those are carried along throughout the code correctly. (Perhaps also call normalize() on each of the dir names returned by this method.)

Reply:
I see fixed.